app/views/directives/add-secret-to-application.html

```diff
@@ -69,6 +69,23 @@
               </div>
             </div>
           </div>
+          <div ng-if="ctrl.addType === 'volume' && ctrl.application.spec.template.spec.containers.length > 1"  class="volume-options">
```

Comment:
We should go ahead do the same for environment variables as well I think

Reply:
Ah, OK

Reply:
Done